bundles/org.openhab.binding.sleepiq/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -126,5 +137,71 @@
 		<description>A detailed message describing an alert condition with the chamber</description>
 		<state readOnly="true"/>
 	</channel-type>
+	<channel-type id="firstNameType">
+		<item-type>String</item-type>
+		<label>First Name</label>
+		<description>The first name of the sleeper</description>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="sleepGoalMinutesType">
+		<item-type>Number</item-type>
+		<label>Sleep Goal</label>
+		<description>The goal for the amount of sleep per night (in minutes)</description>
+		<state readOnly="true" pattern="%d"/>
+	</channel-type>
+	<channel-type id="privacyMode">
+		<item-type>Switch</item-type>
+		<label>Privacy Mode</label>
+		<description>Enable/disable privacy mode</description>
+		<state readOnly="false"/>
+	</channel-type>
+	<channel-type id="todaySleepIQType">
+		<item-type>Number</item-type>
+		<label>Today's Sleep IQ</label>
+		<description>The Sleep Quotient for today</description>
+		<state readOnly="true" pattern="%d"/>
+	</channel-type>
+	<channel-type id="todayAverageHeartRateType">
+		<item-type>Number</item-type>
+		<label>Today's Avg Heart Rate</label>
+		<description>The average heart rate for today</description>
+		<state readOnly="true" pattern="%d"/>
+	</channel-type>
+	<channel-type id="todayAverageRespirationRateType">
+		<item-type>Number</item-type>
+		<label>Today's Avg Respiration Rate</label>
+		<description>The average respiration rate for today</description>
+		<state readOnly="true" pattern="%d"/>
+	</channel-type>
+	<channel-type id="todayMessageType">
+		<item-type>String</item-type>
+		<label>Today's Message</label>
+		<description>The description of today's sleep quality</description>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="todaySleepDurationSecondsType">
+		<item-type>Number</item-type>
```

Comment:
Could this be `Number:Time` for UoM support (and updated as `QuantityType`)?

Reply:
Yes